catalog/dags/providers/provider_api_scripts/nypl.py

```diff
@@ -221,6 +222,22 @@ def _get_creators(creatorinfo):
                 return info.get("namePart", {}).get("$")
         return None
 
+    @staticmethod
+    def _get_tags(mods: dict) -> list[str]:
+        subject_list = mods.get("subject", [])
+        if isinstance(subject_list, dict):
+            subject_list = [subject_list]
```

Comment:
Having a unit test specifically for this new method would be nice.

Reply:
Added in [695bd7d](https://github.com/WordPress/openverse/pull/3774/commits/695bd7d162cddf0c7760edb4ff127f2c34951a36)!